### DISCUSSION TITLE
“Key value” shouldn’t always be hyphenated

### DISCUSSION THREAD

**Discussion by parkerbxyz** [sapegin/textlint-rule-terminology#68] [Bugs]
https://github.com/sapegin/textlint-rule-terminology/blob/afad69ef4e87658d5b3174edbb1552a180ce0306/terms.jsonc#L166

Example of when “key value” should not be hyphenated:

> Key value drivers include the knowledge, skills, experience, training, and creative abilities employees bring to a business and the health of its company culture.

**Comment by sapegin** [comment]
Makes sense! Feel free to send a pull request 👾
